packages/components/card/stories/card.happo.js

```diff
@@ -0,0 +1,57 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '@tradeshift/elements.card';
+
+import { sizes, orientations, types } from '../src/utils';
+
+storiesOf('ts-card', module).add('Simple', () => {
+	const properties = { types, sizes, orientations };
+	const propKeys = Object.keys(properties);
+
+	let templates = [];
+	const content =
+		'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam dolorem ex iste laborum perspiciatis. Impedit ipsa velit voluptatum. A cum, dolore fugiat iure magni nam natus odio quibusdam quod recusandae.';
+
+	propKeys.forEach(propKey => {
+		const propTemplates = [];
+		Object.keys(properties[propKey]).forEach(key => {
+			propTemplates.push(
+				html`
+					<ts-card ${propKey}="${properties[propKey][key]}">${content}</ts-card>
+				`
+			);
+			propKeys.forEach(otherPropKey => {
+				if (otherPropKey !== propKey) {
+					Object.keys(properties[otherPropKey]).forEach(otherKey => {
+						propTemplates.push(
+							html`
+								<ts-card
+									${propKey}="${properties[propKey][key]}"
+									${otherPropKey}="${properties[otherPropKey][otherKey]}"
+									>${content}</ts-card
+								>
+							`
+						);
+					});
+				}
+			});
+		});
+
+		templates.push({
+			prop: propKey,
+			templates: propTemplates
+		});
+	});
+
+	return html`
+		${templates.map(item => {
+			// console.log('templates, prop', templates, prop);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACommentedCode)